### PR TITLE
Updated HQT as the FP8 in llama README

### PR DESCRIPTION
**Pull request by MrGeva**
# Updated LLAMA README to use HQT in FP8

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

### CODE REVIEW

File: examples/text-generation/README.md [hunk 1]
```diff
@@ -218,28 +218,60 @@ python run_generation.py \
...
 --use_hpu_graphs \
+--trim_logits \
 --use_kv_cache \
 --reuse_cache \
---trim_logits \
+--bf16 \
+--batch_size 1
+```
+
+Use this command to quantize the model based on the measurements and run the 70B model:
```
Comment (oabramovich) [inline examples/text-generation/README.md:239]: "based on the measurements from the previous command"

Comment (oabramovich) [reply examples/text-generation/README.md:239]: mention that this runs the model on the full dataset and checks accuracy.

File: examples/text-generation/README.md [hunk 2]
```diff
@@ -218,28 +218,60 @@ python run_generation.py \
...
+--use_hpu_graphs \
+--trim_logits \
+--use_kv_cache \
+--reuse_cache \
+--bf16 \
+--batch_size 1 \
 --fp8
 ```
 
+Use this command to quantize based on the measurements and run batch size 277,
```
Comment (oabramovich) [inline examples/text-generation/README.md:255]: Alternatively, use this command...

**Review by oabramovich — CHANGES_REQUESTED** [review]

**Review by oabramovich — APPROVED** [review]

File: examples/text-generation/README.md [hunk 3]
```diff
@@ -218,28 +218,61 @@ python run_generation.py \
...
+--batch_size 1 \
 --fp8
 ```
 
+Alternatively, use this command to quantize based on the measurements from the previous command. 
+It runs batch size 277, max_new_tokens and max_input_tokens 2048 on the 70B model and prints performance statistics:
+
+```bash
+QUANT_CONFIG=./quantization_config//maxabs_quant.json deepspeed --num_gpus 8 run_generation.py \
+--model_name_or_path meta-llama/Llama-2-70b-hf \
```
Comment (libinta) [inline examples/text-generation/README.md:261]: can you change to
QUANT_CONFIG=./quantization_config/maxabs_measure.json python ../gaudi_spawn.py
--use_deepspeed --world_size 8 run_generation.py

File: examples/text-generation/README.md [hunk 4]
```diff
@@ -218,28 +218,61 @@ python run_generation.py \
...
 ### Running with FP8
 
-Llama2-7b in FP8 is enabled. Use `--fp8` to enable quantization in fp8.
-Add the `--kv_cache_fp8` argument to run the model with a KV cache allocated in fp8.
-More information on enabling fp8 in SynapseAI is available here:
-https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html
+Llama2-70b and 7b in FP8 is enabled using the Quantization Toolkit (HQT), which provides model measurement and quantization capabilities in PyTorch.
+For full details on HQT see: https://docs.habana.ai/en/v1.14.0/PyTorch/Inference_on_PyTorch/Inference_Using_HQT.html
+
```
Comment (libinta) [inline examples/text-generation/README.md:223]: can you change v1.14.0 to latest ?

**Pull request closed without merging**